chat/src/test/java/SpaceManageMessageTestIT.java

```diff
@@ -0,0 +1,172 @@
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceSettingManagement;
+
+@Tag("smoke")
+@Tag("chat")
+public class SpaceManageMessageTestIT extends Base {
+  HomePagePlatform       homePagePlatform;
+
+  NavigationToolbar      navigationToolbar;
+
+  UserAddManagement      userAddManagement;
+
+  ManageLogInOut         manageLogInOut;
+
+  UserAndGroupManagement userandgroupmanagement;
+
+  ChatManagement         chatManagement;
+
+  SpaceManagement        spaceManagement;
+
+  SpaceHomePage          spaceHomePage;
+
+  SpaceSettingManagement spaceSettingManagement;
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+    info("Start setUpBeforeMethod");
+    homePagePlatform = new HomePagePlatform(this);
+    navigationToolbar = new NavigationToolbar(this);
+    userAddManagement = new UserAddManagement(this);
+    manageLogInOut = new ManageLogInOut(this);
+    userandgroupmanagement = new UserAndGroupManagement(this);
+    chatManagement = new ChatManagement(this);
+    spaceManagement = new SpaceManagement(this);
+    spaceHomePage = new SpaceHomePage(this);
+    spaceSettingManagement = new SpaceSettingManagement(this);
+  }
+
+  @Test
+  public void test01_SendMessageOnSpaceChat() {
+    String space = "space" + getRandomNumber();
+    String usernamea = "usernamea" + getRandomString();
+    String password = "123456";
+    String emaila = usernamea + "@test.com";
+    String message = "message" + getRandomNumber();
+    info("Create new user");
+    navigationToolbar.goToAddUser();
+    userAddManagement.addUser(usernamea, password, emaila, usernamea, usernamea);
+    info("Create a space");
+    homePagePlatform.goToHomePage();
+    homePagePlatform.goToMySpaces();
+    spaceManagement.addNewSpaceSimple(space, space);
+    info("connect with user ");
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToAllSpace();
+    spaceManagement.goToAllSpacesTab();
+    info("send request");
+    spaceManagement.sendARequestToASpace(space);
+    homePagePlatform.goToHomePage();
+    manageLogInOut.signIn("root", "gtn");
+    homePagePlatform.goToSpecificSpace(space);
+    spaceHomePage.goToSpaceSettingTab();
+    info("accept request by user 1");
+    spaceSettingManagement.goToMemberTabInSpaceSettingTab();
+    $(byText(usernamea + " " + usernamea)).parent().find(ELEMENT_ICON_ACCEPT_SPACE_REQUEST_IN_MEMBERS_TAB).click();
+    homePagePlatform.goToHomePage();
+    homePagePlatform.goToChat();
+    switchTo().window("Chat");
+    info("send message on space");
+    chatManagement.sendMessageInRoomOrSpace(space, message);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToChat();
+    switchTo().window("Chat");
+    $(byText(space)).click();
+    info("verify message");
+    ELEMENT_CHAT_LIST_MSG.find(byText(message)).should(Condition.exist);
+    switchTo().window("Home Page");
+    manageLogInOut.signIn("root", "gtn");

```

Comment:
don't use strings to sign in, if the default root user is updated, this test will fail.